src/main/java/com/faforever/client/play/PlayController.java

```diff
@@ -51,7 +63,7 @@ public void initialize() {
       } else if (newValue == coopTab) {
         eventBus.post(new OpenCoopEvent());
       }
-    });
+    });*/
```

Comment:
No commented out code... even if u did not add it

Reply:
Yes, I just commited the current status. The whole file is pretty broken right now.